util/testutil/integration/run_windows.go

```diff
@@ -0,0 +1,15 @@
+package integration
+
+var BusyboxImage = "registry.k8s.io/e2e-test-images/busybox:1.29-2"
+
+func officialImages(_ ...string) map[string]string {
+	// supplied string will be ignored, intended for UNIX.
+	// basic mapping for now since there are
+	// very few official Windows-based images.
+	m := map[string]string{
```

Comment:
The names of these images should be defined in the test package like for linux. `util/testutil` should only have the remapping logic etc.

Reply:
moved.